src/main/java/az/subscriptionMS/config/RabbitMQConfig.java

```diff
@@ -0,0 +1,14 @@
+package az.subscriptionMS.config;
```

Comment:
change package name to az.cybernet.ms.subscription

Reply:
OK